### ISSUE TITLE
MiKtex not recongnised

### ISSUE THREAD

**Issue by BeerStud** [texstudio-org/texstudio#2585]
### Environment
- TeXstudio: TeXstudio 4.3.1 (git 4.3.1)
- Qt: Using Qt Version 6.3.1, compiled with Qt 6.3.1 R
- OS: Windows 11 PRO
- TeX distribution: miktex

### Expected behavior
Find the MiKtex distribution 

### Actual behavior
MiKtex not found. See screenshot attached.
![2022-09-16](https://user-images.githubusercontent.com/17906160/190734985-1b2f901d-3c65-44d6-9300-b2d3dbbe1e4f.png)
[System Report.txt](https://github.com/texstudio-org/texstudio/files/9587246/System.Report.txt)

### How to reproduce
Start the program

**Comment by sunderme** [comment]
miktex detection usually works. So the question is why it does not on your system.
Where is it installed ? Is %PATH% not set to miktex ?

**Comment by BeerStud** [comment]
> 
This is where I set the location for the MiKTeX distribution: C:\Users\gerar\AppData\Local\Programs\MiKTeX

My system report:
[System Report.txt](https://github.com/texstudio-org/texstudio/files/9591301/System.Report.txt)

**Comment by sunderme** [comment]
you can either define the location in %PATH% which would make it know to cmd.exe as well or you can set it in txs additional search path.
<img width="1161" alt="grafik" src="https://user-images.githubusercontent.com/14033169/190855845-b2aeddde-9ce8-48f2-9357-15d27b3cf8b8.png">

**Comment by hair-splitter** [comment]
I think this is the problem: 
[https://github.com/MiKTeX/miktex/issues/1159](https://github.com/MiKTeX/miktex/issues/1159)
[https://github.com/MiKTeX/miktex/issues/1183](https://github.com/MiKTeX/miktex/issues/1183)

**Comment by sunderme** [comment]
Any feedback if txs is at fault or miktex removed the binaries ?

**Comment by BeerStud** [comment]
I am at a loss as to how to proceed. I tried removing MiKTex and reinstalling it without success. I even removed TeXstudio and reinstalled it, again without success. This is highly annoying. Is there any movement on this issue? Exactly what file(s) is TeXstudio looking for to verify the existence of the MiKTeX installation? This is all happening on a Windows 11 Pro machine.

**Comment by hair-splitter** [comment]
By looking at the System Report.txt, the TeXstudio looks for the following file on your system:
C:\Users\gerar\AppData\Local\Programs\MiKTeX\miktex\bin\x64\pdflatex.exe

What method do you use to install the MiKTeX?

**Comment by hair-splitter** [comment]
TeXWorks uses miktex-pdftex.exe under MiKTeX, not pdflatex.exe! So even though TeXWorks is working, pdflatex.exe may still be missing.

**Comment by BeerStud** [comment]
I wanted to try and find out what the problem was. So, I used the Revo Uninstaller program to remove both MiKTeX and TeXstudio. I deleted the original downloaded files, cleaned all temp directories, and rebooted the system. Next, I downloaded the MiKTeX Net Installer and ran that. It created a directory filled with hundreds of files. I did not yet attempt to do the actual installation of MiKTeX. I checked for the existence of both miktex-pdftex.exe and pdflatex.exe. Neither one existed; however, pdflatexpicscale.tar.lzma was present.

I am not sure what my next move is. Should I move this over to the MiKTeX support board? It looks like this might be their problem.

**Comment by octaeder** [comment]
I had to install miktex and more a dozen times in the win10 pro sandbox, so I could build Txs. To speedup setup process, I made several notes. Here's what I have for miktex:

MiKTeX: goto https://miktex.org/download where you immediately find basic-miktex-22.7-x64.exe. Run and choose following options:
	1. Install for me (very important)
	2. I choose "installe packages on the fly", go on, do not donate.
	3. After running miktex console we see an "error" message. Click Ok, then under Updates click "Update now".
	4. This takes a while ...
	5. Close the console when you are asked to do so.
TeXstudio: install it after miktex.

Output of echo %PATH% should contain C:\Users\gerar\AppData\Local\Programs\MiKTeX 2.9\miktex\bin\x64\. When I search this directory on my maschine I find following:

![image](https://user-images.githubusercontent.com/102688820/193064499-7adfb084-942f-4f7e-859a-1598cefbc789.png)
